tests/std/tests/VSO_0157762_feature_test_macros/test.compile.pass.cpp

```diff
@@ -1353,6 +1353,20 @@ STATIC_ASSERT(__cpp_lib_smart_ptr_for_overwrite == 202002L);
 #endif
 #endif
 
+#if _HAS_CXX20 && defined(__cpp_consteval)
+#ifndef __cpp_lib_source_location
+#error __cpp_lib_source_location is not defined
+#elif __cpp_lib_source_location != 201907L
+#error __cpp_lib_source_location is not 201907L
+#else

```

Comment:
rewriting this like
 #else //^^^__cpp_lib_source_location / __cpp_lib_source_location vvv 
 will help this make this more readable.
 ditto on other elses elif and endif

Reply:
The lack of comments in the feature-test macro test is intentionally different from the rest of the codebase; basically these blocks of preprocessor logic are dense and compact, so the comments don’t help clarity they way they do in normal code.